src/cron/index.ts

```diff
@@ -13,6 +13,7 @@ import updateHighlightedViews from './updateHighlightedViews';
 import hourlyNotifications from './hourlyNotifications';
 import updateCurrentStreak from './updateCurrentStreak';
 import syncSubscriptionWithCIO from './syncSubscriptionWithCIO';
+// import validateActiveUsers from './validateActiveUsers';
```

Comment:
You don't have to comment here. It will not be triggered anyway since it's commented on pulumi side